fe/fe-core/src/main/java/com/starrocks/task/AgentBatchTask.java

```diff
@@ -349,6 +353,17 @@ private TAgentTaskRequest toAgentTaskRequest(AgentTask task) {
                 tAgentTaskRequest.setUpdate_tablet_meta_info_req(request);
                 return tAgentTaskRequest;
             }
+            case DROP_AUTO_INCREMENT_MAP: {
```

Comment:
Why to DROP this map?

Reply:
If we drop a table, it is not necessary to keep the cached id in BE and FE. This task is for dropping the cached id in BE when we drop a table.

Reply:
So the map is maintained in the table granularity? Currently, we will regard the materialized view as a table.
Does the materialized view support the auto increment column?

Reply:
Yes, in BE, id cached map is maintained in table granularity. The materialized view can be contain auto increment column and because materialized view can not be modify by the user, so auto increment attribute has no effect on it. Drop a materialized view will not trigger the DROP_AUTO_INCREMENT_MAP task.